src/query/xasl_to_stream.c

```diff
@@ -5866,6 +5975,11 @@ xts_sizeof_xasl_node (const XASL_NODE * xasl)
       size += xts_sizeof_access_spec_type (access_spec);
     }
 
+  size += OR_INT_SIZE * 8;	/* cte_xasl_id = sha1(5) + flag(1) + time(2)... */
```

Comment:
In case of cte_xasl_id is NULL?
```suggestion
if (xasl->cte_xasl_id == NULL)
    {
  size += OR_INT_SIZE;
    }
else
    {
  size += OR_INT_SIZE * 8;	/* cte_xasl_id = sha1(5) + flag(1) + time(2)... */
    }
```

Reply:
I fixed the size for "cte_xasl_id" as like handling of list_id.